runtime/src/gpu/cpu/gpu-cpu.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2020-2023 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.  *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef HAS_GPU_LOCALE
+
+#include "sys_basic.h"
+#include "chplrt.h"
+#include "chpl-mem.h"
+#include "chpl-mem-array.h"
+#include "chpl-gpu.h"
+#include "chpl-gpu-impl.h"
+#include "chpl-linefile-support.h"
+#include "chpl-tasks.h"
+#include "error.h"
+#include "chplcgfns.h"
+
+#include <assert.h>
+#include <stdbool.h>
+
+
+void chpl_gpu_impl_init(int* num_devices) {
+  CHPL_GPU_DEBUG("Initializing none GPU layer.\n");
+  *num_devices = 1;
+}
+
+static bool chpl_gpu_device_alloc = false;
+
+void chpl_gpu_impl_on_std_modules_finished_initializing(void) {
+  // The standard module has some memory that we allocate when we  are "on" a
```

Comment:
This is called from a non-impl `chpl_gpu_on_std_modules_finished_initializing` function right?

I think I would just move the comment there are delete it in all of our _impl functions so we don't have this comment duplicated multiple times (too much risk it'll be modified in one copy and not another).